recipes/pytexit/meta.yaml

```
@@ -0,0 +1,43 @@
+{% set name = "pytexit" %}
+{% set version = "0.4.0" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/erwanp/pytexit/archive/{{ version }}.tar.gz
+  sha256: 3fa9f5010ee212bef1b0cb0e2b8383473b7f76144a343e97140fb751e463e7dd
+
+build:
+  noarch: python
+  number: 0
+  script: python -m pip install . --no-deps -vv
```

Please use grayskull to generate Python recipes. Grayskull make it easier for us to review.

See https://github.com/conda-forge/staged-recipes?tab=readme-ov-file#grayskull---recipe-generator-for-python-packages-on-pypi for more information. 

```suggestion
  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
```